apps/dashboard/app/javascript/turbo_shim.js

```diff
@@ -5,21 +5,26 @@
   this shim until we enable it.
 */
 
-import { setInnerHTML } from './utils';
+import { setInnerHTML, restoreFocus } from './utils';
 import { alert } from './alert';
 
 export function replaceHTML(id, html) {
```

Comment:
I'd prefer we keep this method the same and set/restore focus in some other layer. Maybe check if it's focused, replace it, then restore the focus.

It's just like a function should do just 1 thing and 1 thing only sort of idea. Now this function is replacing the HTML AND setting focus, and I'd like to push that somewhere else.